site/src/components/Workspace/Workspace.tsx

```diff
@@ -104,8 +106,15 @@ export const Workspace: FC<React.PropsWithChildren<WorkspaceProps>> = ({
         }
       >
         <WorkspaceStatusBadge build={workspace.latest_build} className={styles.statusBadge} />
-        <PageHeaderTitle>{workspace.name}</PageHeaderTitle>
-        <PageHeaderSubtitle>{workspace.owner_name}</PageHeaderSubtitle>
+        <Box display="flex">
+          {hasTemplateIcon && (
+            <img alt="" src={workspace.template_icon} className={styles.templateIcon} />
```

Comment:
I think for now it's okay to use a generic alt string
```suggestion
            <img alt="Template icon" src={workspace.template_icon} className={styles.templateIcon} />
```